### PR TITLE
Add documentation for feature in #92

### PR DESCRIPTION
**Pull request by Rudd-O**
https://github.com/plone/plone-backend/pull/92/files related.

### CODE REVIEW

File: docs/install/containers/images/backend.md [hunk 1]
```diff
@@ -65,6 +65,13 @@ Plone 6 example:
 docker run -p 8081:8081 -e LISTEN_PORT=8081 plone/plone-backend:{PLONE_BACKEND_VERSION}
 ```
 
+#### Adding configuration to `zope.conf` or additional ZCML
+
+With the standard container, it is not possible to add configuration fragments to
+`zope.conf` directly or add extra ZCML, like it is with the `buildout` deployment
+method.  However, you can derive your own container image, and drop in configuration
+fragments.  See *Extending from this image* below for instructions.
```
Comment (stevepiercy) [inline docs/install/containers/images/backend.md]: Change this to a `ref`.

```suggestion
fragments.  See {ref}`backend-extending-from-this-image-label` below for instructions.
```

Also above the section heading below, insert a label.

```
(backend-extending-from-this-image-label)=

## Extending from this image
```

File: docs/install/containers/images/backend.md [hunk 2]
```diff
@@ -304,6 +311,34 @@ And start a container.
 docker run -p 8080:8080 myproject:latest start
 ```
 
+### Adding `zope.conf` configuration fragments
+
+In the directory containing your `Dockerfile`, create a folder `etc/zope.conf.d`.
+Add your `zope.conf` configuration fragments there.
+
+Now add the following to your `Dockerfile`:
```
Comment (stevepiercy) [inline docs/install/containers/images/backend.md]: Where should it be added? Without guidance, some people will place it in an unfortunate point in the Dockerfile and not take advantage of skipping over steps that have not changed.

File: docs/install/containers/images/backend.md [hunk 3]
```diff
@@ -304,6 +311,34 @@ And start a container.
...
+### Adding ZCML fragments
+
+In the directory containing your `Dockerfile`, create a folder `etc/package-includes`.
+Add your ZCML configuration fragments (named `*-meta.zcml`, `*-configure.zcml`,
+`*-overrides.zcml`) as files in that folder.
+
+Now add the following to your `Dockerfile`:
+
+```Dockerfile
+COPY /etc/package-includes/*.zcml /app/etc/package-includes/
```
Comment (stevepiercy) [inline docs/install/containers/images/backend.md:369]: Where should this be placed in the Dockerfile?

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor enhancement, and questions about where to place commands inside Dockerfile.

File: docs/install/containers/images/backend.md (hunk 2 above)
Comment (Rudd-O) [reply docs/install/containers/images/backend.md]: Good point.  Will fix.

File: docs/install/containers/images/backend.md (hunk 3 above)
Comment (Rudd-O) [reply docs/install/containers/images/backend.md:369]: Good point.  Will fix.

File: docs/install/containers/images/backend.md (hunk 1 above)
Comment (Rudd-O) [reply docs/install/containers/images/backend.md]: It's done.

Comment (stevepiercy) [reply docs/install/containers/images/backend.md]: I don't see the label inserted on line 288. Did you push the commit?

Comment (Rudd-O) [reply docs/install/containers/images/backend.md]: I merged it directly via Github.  WTF.

Comment (stevepiercy) [reply docs/install/containers/images/backend.md]: No, that line you had do separately. I was not able to insert a suggestion in something you had not edited. /me shakes fist at GitHub UI not doing everything under the sun.

**Comment by Rudd-O** [comment]
The ref should be there now @stevepiercy .

**Review by stevepiercy — DISMISSED** [review]
There we go!

**Comment by stevepiercy** [comment]
I'm not sure why the test check failed. I restarted it. Fingers crossed.

**Comment by stevepiercy** [comment]
Passing now. Will merge when https://github.com/plone/plone-backend/pull/92 is merged. Please ping me when ready.

**Comment by Rudd-O** [comment]
@stevepiercy I am ready.  #92 in plone-backend is not yet approved tho.

**Review by stevepiercy — APPROVED** [review]

**Comment by Rudd-O** [comment]
It says "merge blocked. @stevepiercy 

image

**Comment by stevepiercy** [comment]
@Rudd-O correct. Only members of @plone/documentation-team and Organization administrators, repository administrators, and users with the Maintain role, have permission to merge. Once https://github.com/plone/plone-backend/pull/92 is merged, then ping me, and I will merge this one. Or are they waiting for me to merge this one before they merge that one?

**Pull request merged**